From 90c146a7e2d7f902dca82a2c20d531c7512ee988 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Tue, 20 Feb 2018 14:17:51 +0100
Subject: [PATCH] Disable tests and doc

---
 projects.pro | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/projects.pro b/projects.pro
index ba85c995..d2ae9e12 100644
--- a/projects.pro
+++ b/projects.pro
@@ -19,10 +19,7 @@ isEqual( IN_PWD, $${OUT_PWD} ) {
 SUBDIRS += \
     src \
     benchmarks \
-    tests \

-# Docs are always explicitly built with "make doc"
-include(doc/doc.pri)

 check.target = check
 check.CONFIG = recursive
