From 9fc0e12131816cc73b12cb6050ecd570f4bdc243 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Tue, 20 Feb 2018 14:17:51 +0100
Subject: [PATCH] Disable tests and doc

---
 projects.pro | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/projects.pro b/projects.pro
index 478c5840..a4950cdc 100644
--- a/projects.pro
+++ b/projects.pro
@@ -24,10 +24,7 @@ SUBDIRS += \
     src \
     src/icu-extradata \
     benchmarks \
-    tests \
 
-# Docs are always explicitly built with "make doc"
-include(doc/doc.pri)
 
 check.target = check
 check.CONFIG = recursive
-- 
2.14.3

