From 5efb61a37ffb71a563e17d2189784a474f7d7ac9 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Tue, 20 Feb 2018 14:31:55 +0100
Subject: [PATCH] configure: Use a /usr/ prefix by default

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index 316a4153..571179af 100755
--- a/configure
+++ b/configure
@@ -71,7 +71,7 @@ OPT_SHADOW=maybe
 IS_ARMEL=no
    
 # initalize variables used for installation
-M_INSTALL_PREFIX=/usr/local
+M_INSTALL_PREFIX=/usr
 M_INSTALL_SYSCONFDIR=/etc
 M_INSTALL_LIBDIR= # Will be $M_INSTALL_PREFIX/lib unless overriden
 
-- 
2.14.3

