From 2a1623c8b8b99fcd29acf8f555521f7bbdcd2df2 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sat, 21 Oct 2017 22:13:54 +0200
Subject: [PATCH] ffmemless: Reserve enough space for sprintf

---
 src/plugins/ffmemless/ffmemless.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/ffmemless/ffmemless.c b/src/plugins/ffmemless/ffmemless.c
index d92af61..fd3cac0 100644
--- a/src/plugins/ffmemless/ffmemless.c
+++ b/src/plugins/ffmemless/ffmemless.c
@@ -101,7 +101,7 @@ int ffmemless_evdev_file_search(void)
 {
 	int result, i = 0;
 	int fp = 1;
-	char device_file_name[24];
+	char device_file_name[27];
 	unsigned long features[4];

 	/* fail safe stop at 256 devices */
--
2.14.2
