From a83978634cb2a4453df52d4a076dd0d06f86756e Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sat, 22 Apr 2023 18:28:50 +0200
Subject: [PATCH] ffmemless: Reserve enough space for sprintf
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate

---
 src/plugins/ffmemless/ffmemless.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/ffmemless/ffmemless.c b/src/plugins/ffmemless/ffmemless.c
index 9c0cbcf..4141c9b 100644
--- a/src/plugins/ffmemless/ffmemless.c
+++ b/src/plugins/ffmemless/ffmemless.c
@@ -91,7 +91,7 @@ int ffmemless_evdev_file_search(unsigned long features[4])
 	int result;
 	short i = 0;
 	int fp = 1;
-	char device_file_name[24];
+	char device_file_name[27];
 
 	/* fail safe stop at 256 devices */
 	while (fp && i < 256) {
