From 04b2fe6c1f2c416965723a7736999f7d2660fdd7 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Mon, 23 Jan 2017 00:25:55 +0100
Subject: [PATCH] Use pa_card_put and pa_card_choose_initial_profile

---
 src/droid/module-droid-card.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/droid/module-droid-card.c b/src/droid/module-droid-card.c
index 03a5875..bede5eb 100644
--- a/src/droid/module-droid-card.c
+++ b/src/droid/module-droid-card.c
@@ -925,6 +925,9 @@ int pa__init(pa_module *m) {
     u->card->userdata = u;
     u->card->set_profile = card_set_profile;
 
+    pa_card_choose_initial_profile(u->card);
+    pa_card_put(u->card);
+
     u->modargs = ma;
     u->module = m;
 
-- 
2.11.0

