From 1e170f447d7b8b02dad98912d521142a7fff8c30 Mon Sep 17 00:00:00 2001
From: casept <davids.paskevics@gmail.com>
Date: Thu, 26 Feb 2026 12:57:19 +0100
Subject: [PATCH] Remove audio sources that don't exist for some reason

Upstream-Status: Inappropriate

---
 src/common/droid-util-audio.h | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/common/droid-util-audio.h b/src/common/droid-util-audio.h
index 88322c3..5c659e2 100644
--- a/src/common/droid-util-audio.h
+++ b/src/common/droid-util-audio.h
@@ -120,8 +120,6 @@ uint32_t conversion_table_default_audio_source[][2] = {
     { AUDIO_DEVICE_IN_PROXY,                        AUDIO_SOURCE_MIC                        },
     { AUDIO_DEVICE_IN_USB_HEADSET,                  AUDIO_SOURCE_MIC                        },
     { AUDIO_DEVICE_IN_BLUETOOTH_BLE,                AUDIO_SOURCE_MIC                        },
-    { AUDIO_DEVICE_IN_HDMI_ARC,                     AUDIO_SOURCE_MIC                        },
-    { AUDIO_DEVICE_IN_ECHO_REFERENCE,               AUDIO_SOURCE_MIC                        },
 
 #if defined(HAVE_ENUM_AUDIO_DEVICE_IN_FM_RX) && defined(HAVE_ENUM_AUDIO_SOURCE_FM_RX)
     { AUDIO_DEVICE_IN_FM_RX,                        AUDIO_SOURCE_FM_RX                      },
@@ -174,7 +172,6 @@ struct string_conversion string_conversion_table_audio_mode_fancy[] = {
     { AUDIO_MODE_RINGTONE,                              "ringtone"                          },
     { AUDIO_MODE_IN_CALL,                               "in call"                           },
     { AUDIO_MODE_IN_COMMUNICATION,                      "in communication"                  },
-    { AUDIO_MODE_CALL_SCREEN,                           "call screen"                       },
 
     { 0, NULL }
 };
@@ -244,8 +241,6 @@ struct string_conversion string_conversion_table_input_device[] = {
     STRING_ENTRY( AUDIO_DEVICE_IN_PROXY                             ),
     STRING_ENTRY( AUDIO_DEVICE_IN_USB_HEADSET                       ),
     STRING_ENTRY( AUDIO_DEVICE_IN_BLUETOOTH_BLE                     ),
-    STRING_ENTRY( AUDIO_DEVICE_IN_HDMI_ARC                          ),
-    STRING_ENTRY( AUDIO_DEVICE_IN_ECHO_REFERENCE                    ),
     STRING_ENTRY( AUDIO_DEVICE_IN_DEFAULT                           ),
 
     /* Devices which may or may not be defined for all devices. */
@@ -281,8 +276,6 @@ struct string_conversion string_conversion_table_input_device_fancy[] = {
     { AUDIO_DEVICE_IN_PROXY,                            "input-proxy"                       },
     { AUDIO_DEVICE_IN_USB_HEADSET,                      "input-usb_headset"                 },
     { AUDIO_DEVICE_IN_BLUETOOTH_BLE,                    "input-bluetooth_ble"               },
-    { AUDIO_DEVICE_IN_HDMI_ARC,                         "input-hdmi_arc"                    },
-    { AUDIO_DEVICE_IN_ECHO_REFERENCE,                   "input-echo_reference"              },
     { AUDIO_DEVICE_IN_DEFAULT,                          "input-default"                     },
 
     /* Devices which may or may not be defined for all devices. */
@@ -304,7 +297,6 @@ struct string_conversion string_conversion_table_audio_source_fancy[] = {
     { AUDIO_SOURCE_VOICE_COMMUNICATION,                 "voice communication"               },
     { AUDIO_SOURCE_REMOTE_SUBMIX,                       "remote submix"                     },
     { AUDIO_SOURCE_UNPROCESSED,                         "unprocessed"                       },
-    { AUDIO_SOURCE_VOICE_PERFORMANCE,                   "voice performance"                 },
 
     /* Audio sources which may or may not be defined for all devices. */
     FANCY_ENTRY_IF_AUDIO_SOURCE_ECHO_REFERENCE        ( "echo reference"                    )
@@ -349,7 +341,6 @@ struct string_conversion string_conversion_table_input_flag[] = {
     STRING_ENTRY( AUDIO_INPUT_FLAG_MMAP_NOIRQ                       ),
     STRING_ENTRY( AUDIO_INPUT_FLAG_VOIP_TX                          ),
     STRING_ENTRY( AUDIO_INPUT_FLAG_HW_AV_SYNC                       ),
-    STRING_ENTRY( AUDIO_INPUT_FLAG_DIRECT                           ),
 
     { 0, NULL }
 };
