From 6ae25562fb7950755e719060e3642c8a65bb4f5e Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 30 Oct 2016 17:49:03 +0100
Subject: [PATCH] Fix error with GCC 6 caused by an inconsistent usage of
 'auto'

---
 include/cor/error.hpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/cor/error.hpp b/include/cor/error.hpp
index 72fe5d4..645ae75 100644
--- a/include/cor/error.hpp
+++ b/include/cor/error.hpp
@@ -93,7 +93,8 @@ public:
         std::copy(&src.frames[0], &src.frames[count_], frames.begin());
         if (src.symbols) {
             symbols.reset((const char**)calloc(src.count_, sizeof(*symbols)));
-            auto to = src.symbols.get(), from = symbols.get();
+            auto to = src.symbols.get();
+            auto from = symbols.get();
             std::copy(from, from + count_, to);
         }
     }
--
2.10.1
