From b4c1f3a6509a51275054f77de5fd10d1bbecb6ad Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sat, 21 Oct 2017 21:32:54 +0200
Subject: [PATCH] Add a missing header include

---
 include/cor/util.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/cor/util.hpp b/include/cor/util.hpp
index 153c496..f339ce1 100644
--- a/include/cor/util.hpp
+++ b/include/cor/util.hpp
@@ -2,7 +2,7 @@
 #define _COR_UTIL_HPP_
 
 #include <cor/error.hpp>
-
+#include <functional>
 #include <unistd.h>
 
 #include <vector>
-- 
2.14.2

