commit 0a515a50c2aa1ce0f77563a22c719bfd58ecade7
Author: Florent Revest <revestflo@gmail.com>
Date:   Mon Jun 8 14:43:24 2015 +0200

    Disables resourceqt-client build

diff --git a/libresourceqt.pro b/libresourceqt.pro
index 1177ab3..7ba2f54 100644
--- a/libresourceqt.pro
+++ b/libresourceqt.pro
@@ -24,11 +24,9 @@ TEMPLATE = subdirs
 SUBDIRS = libdbus-qeventloop  \
           libresourceqt	      \
           libmediaoverridesqt \
-          resourceqt-client   \
           tests

 tests.depends = libdbus-qeventloop libresourceqt
-resourceqt-client.depends = libdbus-qeventloop libresourceqt


 dist.commands   = ./makedist.sh
