From b6f2c4ecba921f6f685c0c9d1d8838f5a575bd25 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 28 Sep 2017 15:04:02 +0200
Subject: [PATCH] Don't require QtGui

---
 cmake/modules/FindQt5.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmake/modules/FindQt5.cmake b/cmake/modules/FindQt5.cmake
index ffd5dd3..6378f6d 100644
--- a/cmake/modules/FindQt5.cmake
+++ b/cmake/modules/FindQt5.cmake
@@ -66,7 +66,6 @@ find_package(Qt5Core ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5DBus ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Gui ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Test ${REQUIRED_QT_VERSION} REQUIRED)
-find_package(Qt5Widgets ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Network ${REQUIRED_QT_VERSION} REQUIRED)
 find_package(Qt5Xml ${REQUIRED_QT_VERSION} REQUIRED)

--
2.7.4
