From ea4167c12097786f67136828fb68c29b7e894bba Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 28 Sep 2017 15:06:27 +0200
Subject: [PATCH] Don't require python

---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2a74720..068b555 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -136,15 +136,6 @@ macro_log_feature(QT_GLIB_SUPPORT "Qt Glib Support"
 # reset flags
 set(CMAKE_REQUIRED_FLAGS "")

-# Find python version >= 2.7
-set(REQUIRED_PY 2.7)
-find_package(PythonLibrary ${REQUIRED_PY} REQUIRED)
-if(${PYTHON_SHORT_VERSION} VERSION_GREATER ${REQUIRED_PY} OR ${PYTHON_SHORT_VERSION} VERSION_EQUAL ${REQUIRED_PY})
-    message(STATUS "Python ${PYTHON_SHORT_VERSION} found")
-else()
-    message(SEND_ERROR "Python >= ${REQUIRED_PY} is required")
-endif()
-
 # Check for dbus-python
 execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "import dbus.mainloop.glib"
                 RESULT_VARIABLE PYTHON_DBUS_RESULT)
--
2.7.4
