From e1efb911f8fae2fe384ddcb52dc4ffb7744f6e79 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 28 Sep 2017 15:14:26 +0200
Subject: [PATCH] Disable examples compilation

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 068b555..40684c9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -90,7 +90,7 @@ option(ENABLE_SERVICE_SUPPORT "Enable compilation of service side bindings for T
 # Add an option to select type of service library
 option(FORCE_STATIC_SERVICE_LIBRARY "Force Telepathy-Qt service to be a static library." FALSE)
 # Add an option for compiling examples
-option(ENABLE_EXAMPLES "Enable compilation of examples for Telepathy-Qt" TRUE)
+option(ENABLE_EXAMPLES "Enable compilation of examples for Telepathy-Qt" FALSE)
 # Add an option for compiling Farstream
 option(ENABLE_FARSTREAM "Enable compilation of Farstream bindings" TRUE)
 # Add an option for building tests
--
2.7.4
