From 8a0bf3b01ea58f80f448384af7b395703fa9e527 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 26 Aug 2018 23:17:13 +0200
Subject: [PATCH] booster-generic: Fix path to tibapplauncherd

Upstream-Status: Inappropriate

---
 src/booster-generic/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/booster-generic/CMakeLists.txt b/src/booster-generic/CMakeLists.txt
index 2b0672b..1010de9 100644
--- a/src/booster-generic/CMakeLists.txt
+++ b/src/booster-generic/CMakeLists.txt
@@ -10,7 +10,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden")
 set(SRC booster-generic.cpp)
 
 # Set libraries to be linked.
-link_libraries("-L../launcherlib -lapplauncherd" ${LIBDL})
+link_libraries("-Lsrc/launcherlib -lapplauncherd" ${LIBDL})
 
 # Set executable
 add_executable(booster-generic ${SRC} ${MOC_SRC})
