From 646de2a16c53baddf34ac0f03e803854edd496aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Darrel=20Gri=C3=ABt?= <dgriet@gmail.com>
Date: Thu, 26 Dec 2024 21:38:41 +0100
Subject: [PATCH] invoker: Do not return a value for a void function
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate

Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 src/invoker/invoker.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/invoker/invoker.c b/src/invoker/invoker.c
index ba81f7b..4fa3aec 100644
--- a/src/invoker/invoker.c
+++ b/src/invoker/invoker.c
@@ -468,7 +468,6 @@ static void notify_app_lauch(const char *desktop_file)
     } else {
         info("Failed to connect to the DBus session bus: %s", error.message);
         dbus_error_free(&error);
-        return 1;
     }
 }
 
