From e9f6d8790f6a7a89eb75d9c6d480eaba668903ac Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 28 Sep 2017 23:42:23 +0200
Subject: [PATCH] Use libsystemd instead of libsystemd-daemon

---
 src/hostlib/hostlib.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hostlib/hostlib.pro b/src/hostlib/hostlib.pro
index 9cc66f6..88adb94 100644
--- a/src/hostlib/hostlib.pro
+++ b/src/hostlib/hostlib.pro
@@ -15,7 +15,7 @@ QT += dbus
 PKGCONFIG += \
         dbus-1 \
         keepalive \
-        libsystemd-daemon \
+        libsystemd \
         mlite5
 
 INCLUDEPATH += \
-- 
2.7.4

