From b218234958130371232691d636b5c5af1c02ff00 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Tue, 27 Jan 2015 16:09:00 +0100
Subject: [PATCH] Link against libEGL to satisfy runtime dependencies

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 customcontext/customcontext.pro |    1 +
 1 file changed, 1 insertion(+)

diff --git a/customcontext/customcontext.pro b/customcontext/customcontext.pro
index 39c993d..b1dfdd2 100644
--- a/customcontext/customcontext.pro
+++ b/customcontext/customcontext.pro
@@ -96,6 +96,7 @@ hybristexture :{
     SOURCES += texture/hybristexture.cpp
     HEADERS += texture/hybristexture.h
     INCLUDEPATH += texture
+    LIBS += -lEGL
 
 } else {
     message("hybristexture ............: no")
-- 
1.7.9.5

