From dbf1f0b85bcc2343b3a60f461c7251f269450571 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 6 Aug 2015 14:32:10 +0200
Subject: [PATCH] Includes sync.h which provides sync_wait
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate
Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 hwcomposer/hwcomposer_backend_v10.h | 1 +
 hwcomposer/hwcomposer_backend_v11.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/hwcomposer/hwcomposer_backend_v10.h b/hwcomposer/hwcomposer_backend_v10.h
index 0ad3e17..8debfec 100644
--- a/hwcomposer/hwcomposer_backend_v10.h
+++ b/hwcomposer/hwcomposer_backend_v10.h
@@ -53,6 +53,7 @@
 #include <QBasicTimer>
 #include <QSet>
 #include <QWindow>
+#include <sync/sync.h>
 
 class HwComposerBackend_v10 : public QObject, public HwComposerBackend {
 public:
diff --git a/hwcomposer/hwcomposer_backend_v11.h b/hwcomposer/hwcomposer_backend_v11.h
index 2bf91c9..b25dc05 100644
--- a/hwcomposer/hwcomposer_backend_v11.h
+++ b/hwcomposer/hwcomposer_backend_v11.h
@@ -45,6 +45,7 @@
 #ifdef HWC_PLUGIN_HAVE_HWCOMPOSER1_API
 
 #include "hwcomposer_backend.h"
+#include <sync/sync.h>
 
 // libhybris access to the native hwcomposer window
 #include <hwcomposer_window.h>
-- 
2.50.0

