Upstream-Status: Inappropriate

diff --git a/bluebinder.service b/bluebinder.service
index 1ee68f7..7fd42bd 100644
--- a/bluebinder.service
+++ b/bluebinder.service
@@ -1,29 +1,16 @@
 [Unit]
 Description=Simple proxy for using android binder based bluetooth through vhci.
-After=droid-hal-init.service
+After=android-system.service
 Before=bluetooth.service
 
 [Service]
 Type=notify
 EnvironmentFile=-/var/lib/environment/bluebinder/*.conf
-ExecStartPre=/usr/bin/droid/bluebinder_wait.sh
+ExecStartPre=/usr/sbin/bluebinder_wait.sh
 ExecStart=/usr/sbin/bluebinder
-ExecStartPost=/usr/bin/droid/bluebinder_post.sh
 Restart=always
 TimeoutStartSec=60
-# Sandboxing
-CapabilityBoundingSet=CAP_DAC_READ_SEARCH
-DeviceAllow=/dev/binder rw
-DeviceAllow=/dev/hwbinder rw
-DeviceAllow=/dev/vhci rw
-DeviceAllow=/dev/rfkill r
-DevicePolicy=strict
-NoNewPrivileges=yes
-RestrictAddressFamilies=AF_BLUETOOTH
-PrivateTmp=yes
-ProtectHome=yes
-ProtectSystem=full
 
 [Install]
-WantedBy=graphical.target
+WantedBy=multi-user.target
 
diff --git a/bluebinder_wait.sh b/bluebinder_wait.sh
index d7a059d..c3f9f99 100755
--- a/bluebinder_wait.sh
+++ b/bluebinder_wait.sh
@@ -4,9 +4,9 @@ while true
 do
     /usr/bin/getprop | grep -q init.svc.*.bluetooth.audio
     if [ $? -eq 0 ] ; then
-        bt_status=$(/usr/bin/getprop |grep "init.svc.*bluetooth" |grep -v audio |grep -o "\[running\]")
+        bt_status=$(/usr/bin/getprop |grep "init.svc.*bluetooth" |grep -v audio |grep -o "\[running\]" | head -1)
     else
-        bt_status=$(/usr/bin/getprop |grep "init.svc.*bluetooth" |grep -o "\[running\]")
+        bt_status=$(/usr/bin/getprop |grep "init.svc.*bluetooth" |grep -o "\[running\]" | head -1)
     fi
     if [ "$bt_status" = "[running]" ] ; then
         echo "Bluetooth service running"
