diff --git a/bluebinder.service b/bluebinder.service
index 8b63181..52eae10 100644
--- a/bluebinder.service
+++ b/bluebinder.service
@@ -1,27 +1,15 @@
 [Unit]
 Description=Simple proxy for using android binder based bluetooth through vhci.
-After=droid-hal-init.service
+After=android-system.service
 Before=bluetooth.service
 
 [Service]
 Type=notify
-ExecStartPre=/usr/bin/droid/bluebinder_wait.sh
+ExecStartPre=/usr/sbin/bluebinder_wait.sh
 ExecStart=/usr/sbin/bluebinder
-ExecStartPost=/usr/bin/droid/bluebinder_post.sh
 Restart=always
 TimeoutStartSec=60
-# Sandboxing
-CapabilityBoundingSet=CAP_DAC_READ_SEARCH
-DeviceAllow=/dev/hwbinder rw
-DeviceAllow=/dev/vhci rw
-DeviceAllow=/dev/rfkill r
-DevicePolicy=strict
-NoNewPrivileges=yes
-RestrictAddressFamilies=AF_BLUETOOTH
-PrivateTmp=yes
-ProtectHome=yes
-ProtectSystem=full
 
 [Install]
-WantedBy=graphical.target
+WantedBy=multi-user.target
 
