From 1586d7e8e2bd8de1b8ee12924871adf240e831da Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 30 Oct 2016 18:13:39 +0100
Subject: [PATCH] Add a missing math.h include

---
 src/util.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/util.cpp b/src/util.cpp
index d738815..0843785 100644
--- a/src/util.cpp
+++ b/src/util.cpp
@@ -11,6 +11,8 @@
 #include <QString>
 #include <QMap>
 
+#include <math.h>
+
 namespace qtaround {
 
 namespace error {
-- 
2.10.1

