From 7ce14bff26d267d7fc5243025a4150e23feb937d Mon Sep 17 00:00:00 2001
From: MagneFire <IDaNLContact@gmail.com>
Date: Tue, 5 Jan 2021 19:47:40 +0100
Subject: [PATCH] Disable tests. Fixes compilation issues.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f69dd99..7cdb686 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,7 +33,7 @@ set(QTAROUND_MOC_HEADERS
 )
 
 add_subdirectory(src)
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
 configure_file(qtaround.pc.in qtaround.pc @ONLY)
 configure_file(qtaround-1.pc.in qtaround-1.pc @ONLY)
-- 
2.30.0

