From 58707b9ef53379e0eabb943e4448dc973f96126d Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 28 Sep 2017 14:38:13 +0200
Subject: [PATCH] Fixes build

Upstream-Status: Inappropriate

---
 src/src.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/src.pro b/src/src.pro
index f0fa35a..c04b5e9 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -45,5 +45,5 @@ OTHER_FILES += voicecall-manager.desktop voicecall-manager.service
 systemd_service_entry.files = voicecall-manager.service
 systemd_service_entry.path = /usr/lib/systemd/user

-INSTALLS += autostart_entry systemd_service_entry
+INSTALLS += systemd_service_entry

--
2.7.4
