From 4b062c99236f4fd7069aa95d47971a5f63605d26 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 28 Feb 2016 13:45:41 +0100
Subject: [PATCH] Fixes build with OE

---
 droid-vibrator.pro | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/droid-vibrator.pro b/droid-vibrator.pro
index 4f0806a..c55874c 100644
--- a/droid-vibrator.pro
+++ b/droid-vibrator.pro
@@ -11,7 +11,7 @@ SOURCES += qfeedback.cpp
 
 CONFIG += link_pkgconfig
 PKGCONFIG += android-headers libhardware libvibrator
-DEFINES += 'DROID_VIBRATOR_SETTINGS=\'\"$$[QT_INSTALL_PLUGINS]/feedback/droid-vibrator.ini\"\''
+DEFINES += 'DROID_VIBRATOR_SETTINGS=\'\"/usr/lib/plugins/feedback/droid-vibrator.ini\"\''
 
 settings.files = droid-vibrator.ini
 settings.path = $$[QT_INSTALL_PLUGINS]/feedback
@@ -27,4 +27,4 @@ INSTALLS += plugindescription
 # also enable profile detection. libprofile-qt5 is a bit broken, work around it here.
 QT += dbus
 QMAKE_CXXFLAGS += -I/usr/include/profile-qt5
-QMAKE_LFLAGS += -lprofile-qt5
+LIBS += -lprofile-qt5
-- 
2.7.0

