From ec2014d39983c0a879a4190faf792169743a60c3 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Mon, 16 Jan 2017 13:01:39 +0100
Subject: [PATCH 2/2] languagemodel: don't require region names

---
 src/languagemodel.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/languagemodel.cpp b/src/languagemodel.cpp
index 2ea0d72..4b493f4 100644
--- a/src/languagemodel.cpp
+++ b/src/languagemodel.cpp
@@ -222,7 +222,7 @@ QList<Language> LanguageModel::supportedLanguages()
         QString region = settings.value("Region").toString();
         //% "Region: %1"
         QString regionLabel = settings.value("RegionLabel", qtTrId("settings_system-la-region")).toString();
-        if (name.isEmpty() || localeCode.isEmpty() || region.isEmpty()) {
+        if (name.isEmpty() || localeCode.isEmpty()) {
             continue;
         }
         Language newLanguage(name, localeCode, region, regionLabel);
-- 
2.11.0

