From e8756a209aaf4aecf5ad9705f450ddfbb3185f38 Mon Sep 17 00:00:00 2001
From: MagneFire <IDaNLContact@gmail.com>
Date: Tue, 5 Jan 2021 21:26:03 +0100
Subject: [PATCH] meson: Disable docs and tests.

---
 meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/meson.build b/meson.build
index 1933f44..6f9192a 100644
--- a/meson.build
+++ b/meson.build
@@ -36,5 +36,3 @@ root_dir = include_directories('.')
 subdir('libaccounts-glib')
 subdir('tools')
 subdir('data')
-subdir('docs')
-subdir('tests')
--
2.30.0
