From 94bb3dab9317bfab144db0fc75e2169730cecb0d Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Wed, 31 Jul 2019 11:47:08 +0000
Subject: [PATCH] Use QGstTools as a private Qt module instead of a lib

Upstream-Status: Inappropriate

---
 src/videotexturebackend/videotexturebackend.pro | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/videotexturebackend/videotexturebackend.pro b/src/videotexturebackend/videotexturebackend.pro
index 6803c26..61954da 100644
--- a/src/videotexturebackend/videotexturebackend.pro
+++ b/src/videotexturebackend/videotexturebackend.pro
@@ -7,7 +7,8 @@ QT += \
         quick \
         multimedia \
         multimedia-private \
-        qtmultimediaquicktools-private
+        qtmultimediaquicktools-private \
+        multimediagsttools-private

 CONFIG += plugin hide_symbols link_pkgconfig

@@ -16,8 +17,6 @@ PKGCONFIG +=\
         gstreamer-1.0 \
         nemo-gstreamer-interfaces-1.0

-LIBS += -lqgsttools_p
-
 # It won't compile without this,
 # the issue is Xlib.h defines Bool as int but  QJsonValue.h has an enum with Bool = 0x1 --> int = 0x1 -> BOOM!
 DEFINES += MESA_EGL_NO_X11_HEADERS
--
2.20.1
