commit d474b630a62265e2090489a9048f510d3d8d2746
Author: Florent Revest <revestflo@gmail.com>
Date:   Fri Jun 5 13:28:45 2015 +0200

    Allows use of custom CFLAGS

diff --git a/Makefile b/Makefile
index 7aa145a..0738aa9 100644
--- a/Makefile
+++ b/Makefile
@@ -103,17 +103,17 @@ install_tests::
 # Common options
 # ----------------------------------------------------------------------------
 
-CFLAGS   += -std=c99
-CFLAGS   += -O2
-CFLAGS   += -Wall
-CFLAGS   += -Wwrite-strings
-CFLAGS   += -Wmissing-prototypes
-CFLAGS   += -Werror
+override CFLAGS   += -std=c99
+override CFLAGS   += -O2
+override CFLAGS   += -Wall
+override CFLAGS   += -Wwrite-strings
+override CFLAGS   += -Wmissing-prototypes
+override CFLAGS   += -Werror
 
-CFLAGS   += -g
-LDFLAGS  += -g
+override CFLAGS   += -g
+override LDFLAGS  += -g
 
-LDFLAGS  += -Wl,--as-needed
+override LDFLAGS  += -Wl,--as-needed
 
 # ----------------------------------------------------------------------------
 # libdsme$(SOVERS) and libdsme.a
@@ -122,11 +122,11 @@ LDFLAGS  += -Wl,--as-needed
 libdsme_OBJ += protocol.pic.o message.pic.o alarm_limit.pic.o
 libdsme_PC  += glib-2.0
 
-libdsme$(SOVERS) : CFLAGS += $$(pkg-config --cflags $(libdsme_PC))
-libdsme$(SOVERS) : LDLIBS += $$(pkg-config --libs $(libdsme_PC))
+libdsme$(SOVERS) : override CFLAGS += $$(pkg-config --cflags $(libdsme_PC))
+libdsme$(SOVERS) : override LDLIBS += $$(pkg-config --libs $(libdsme_PC))
 libdsme$(SOVERS) : $(libdsme_OBJ)
 
-libdsme.a : CFLAGS += $$(pkg-config --cflags $(libdsme_PC))
+libdsme.a : override CFLAGS += $$(pkg-config --cflags $(libdsme_PC))
 libdsme.a : $(patsubst %.pic.o, %.o, $(libdsme_OBJ))
 
 # ----------------------------------------------------------------------------
@@ -152,6 +152,6 @@ libthermalmanager_dbus_if$(SOVERS) : $(libthermalmanager_dbus_if_OBJ)
 ut_libdsme_OBJ += tests/ut_libdsme.o
 ut_libdsme_PC  += glib-2.0 check
 
-tests/ut_libdsme : CFLAGS += $$(pkg-config --cflags $(ut_libdsme_PC))
-tests/ut_libdsme : LDLIBS += $$(pkg-config --libs $(ut_libdsme_PC))
+tests/ut_libdsme : override CFLAGS += $$(pkg-config --cflags $(ut_libdsme_PC))
+tests/ut_libdsme : override LDLIBS += $$(pkg-config --libs $(ut_libdsme_PC))
 tests/ut_libdsme : $(ut_libdsme_OBJ) libdsme$(SOVERS)
