From d99fd780c41fdcc905792b6fd39757de2ac1252f Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Mon, 17 Aug 2015 12:20:58 +0200
Subject: [PATCH] Don't draw progress and message bar

---
 psplash.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/psplash.c b/psplash.c
index 1a56629..fbfb823 100644
--- a/psplash.c
+++ b/psplash.c
@@ -312,7 +312,7 @@ main (int argc, char** argv)
 			 POKY_IMG_RLE_PIXEL_DATA);
 
   /* Draw progress bar border */
-  psplash_fb_draw_image (fb, 
+/*  psplash_fb_draw_image (fb, 
 			 (fb->width  - BAR_IMG_WIDTH)/2, 
 			 SPLIT_LINE_POS(fb),
 			 BAR_IMG_WIDTH,
@@ -325,7 +325,7 @@ main (int argc, char** argv)
 
 #ifdef PSPLASH_STARTUP_MSG
   psplash_draw_msg (fb, PSPLASH_STARTUP_MSG);
-#endif
+#endif */
 
   /* Scene set so let's flip the buffers. */
   /* The first time we also synchronize the buffers so we can build on an
