From 6b6610ad3e0f84c212231f08d9433e7ba4b3e3f6 Mon Sep 17 00:00:00 2001
From: Richard Barnes <rbarnes@umn.edu>
Date: Wed, 2 Oct 2024 10:35:09 -0700
Subject: [PATCH] Match `malloc` signature to its use

Upstream-Status: Backport bf254c75b1e0d2641ebbd7fc85fb183f36a62ea7
---
 lib/malloc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/malloc.c b/lib/malloc.c
index 75e8ef9..701b9b3 100755
--- a/lib/malloc.c
+++ b/lib/malloc.c
@@ -3,7 +3,7 @@
      
      #include <sys/types.h>
      
-     void *malloc ();
+     void *malloc (size_t n);
      
      /* Allocate an N-byte block of memory from the heap.
         If N is zero, allocate a 1-byte block.  */
