From ef9a6b7b102b2e83757ff7cfca4c17f1f336557b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Darrel=20Gri=C3=ABt?= <dgriet@gmail.com>
Date: Sun, 20 Nov 2022 18:44:31 +0100
Subject: [PATCH] Makefile: Allow for INSTALL_SYSTEMD_DIR to be overridden.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate

Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index bd8b630..ee4f3d0 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -204,7 +204,7 @@ INSTALL = install
 INSTALL_DIRS = $(INSTALL) -d
 
 INSTALL_SBIN_DIR = $(DESTDIR)/usr/sbin
-INSTALL_SYSTEMD_DIR = $(DESTDIR)$(ABS_UNITDIR)
+INSTALL_SYSTEMD_DIR ?= $(DESTDIR)$(ABS_UNITDIR)
 
 install: $(RELEASE_EXE) $(INSTALL_SBIN_DIR) $(INSTALL_SYSTEMD_DIR)
 	$(INSTALL) -m 644 nfcd.service "$(INSTALL_SYSTEMD_DIR)"
