From 31b4416475b0fbf1b11a3d194a9e738de7d20fd0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Darrel=20Gri=C3=ABt?= <dgriet@gmail.com>
Date: Sun, 27 Nov 2022 18:21:43 +0100
Subject: [PATCH] systemd: Allow the service to be started as root.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate

Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 src/nfcd.service | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/nfcd.service b/src/nfcd.service
index 7fc30ad..99db011 100644
--- a/src/nfcd.service
+++ b/src/nfcd.service
@@ -6,7 +6,6 @@ After=dbus.service
 [Service]
 Type=dbus
 BusName=org.sailfishos.nfc.daemon
-User=nfc
 EnvironmentFile=-/var/lib/environment/nfcd/*.conf
 ExecStart=/usr/sbin/nfcd -o syslog $NFCD_ARGS
 Restart=always
