From b383eccb4d6ee099b9cabe1643c223fdc20d815b Mon Sep 17 00:00:00 2001
From: Sergey Chupligin <neochapay@gmail.com>
Date: Mon, 28 Nov 2022 14:53:04 +0300
Subject: [PATCH] [libconnman-qt] Fixup warning on Qt >= 5.10

---
 libconnman-qt/counter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libconnman-qt/counter.cpp b/libconnman-qt/counter.cpp
index 0b95ce6..603c14b 100644
--- a/libconnman-qt/counter.cpp
+++ b/libconnman-qt/counter.cpp
@@ -31,10 +31,10 @@ Counter::Counter(QObject *parent) :
     shouldBeRunning(false),
     registered(false)
 {
-    QTime time = QTime::currentTime();
     #if (QT_VERSION >= QT_VERSION_CHECK(5,10,0))
         quint32 randomValue = QRandomGenerator::global()->generate();
     #else
+        QTime time = QTime::currentTime();
         qsrand((uint)time.msec());
         int randomValue = qrand();
     #endif
