From 22d79a9cf447566c04c50b4bf338668d214e6731 Mon Sep 17 00:00:00 2001
From: MagneFire <IDaNLContact@gmail.com>
Date: Sat, 9 Jan 2021 16:34:06 +0100
Subject: [PATCH] Set version to 13 to support PulseAudio 13.

---
 .tarball-version | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 .tarball-version

diff --git a/.tarball-version b/.tarball-version
new file mode 100644
index 0000000..02161ca
--- /dev/null
+++ b/.tarball-version
@@ -0,0 +1 @@
+13.0.0
-- 
2.30.0

