From 74f0bdde15478783aedc37f3d2130aa9441cac34 Mon Sep 17 00:00:00 2001
From: Davids Paskevics <davids.paskevics@gmail.com>
Date: Thu, 2 Oct 2025 20:05:48 +0200
Subject: [PATCH] Disable tests

Upstream-Status: Inappropriate

---
 keepalive.pro | 1 -
 1 file changed, 1 deletion(-)

diff --git a/keepalive.pro b/keepalive.pro
index 8f564cc..a8b20f2 100644
--- a/keepalive.pro
+++ b/keepalive.pro
@@ -6,7 +6,6 @@ SUBDIRS += plugin
 SUBDIRS += examples/backgroundactivity_periodic
 SUBDIRS += examples/backgroundactivity_linger
 SUBDIRS += examples/displayblanking
-SUBDIRS += tests
 SUBDIRS += doc
 
 examples.files = examples/qml/*.qml
