From 8231557cb3d2ee01382ceae0e76d74358a40afc0 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sat, 12 Dec 2015 18:50:02 +0100
Subject: [PATCH] Forces GLES2 the dirty way

---
 src/platformsupport/eglconvenience/qeglplatformcontext.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/platformsupport/eglconvenience/qeglplatformcontext.cpp b/src/platformsupport/eglconvenience/qeglplatformcontext.cpp
index bffb0eb..cdd482b 100644
--- a/src/platformsupport/eglconvenience/qeglplatformcontext.cpp
+++ b/src/platformsupport/eglconvenience/qeglplatformcontext.cpp
@@ -306,6 +306,8 @@ void QEGLPlatformContext::updateFormatFromGL()
                         minor = 0;
                     }
 #endif
+                    major = 2;
+                    minor = 0;
                     m_format.setMajorVersion(major);
                     m_format.setMinorVersion(minor);
                 }
-- 
2.6.2
