From d87264156feedf8b194967c6758d1eea43e33c33 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Fri, 8 Sep 2017 14:52:31 +0200
Subject: [PATCH] configure.ac: Check hardfp from cross compilation toolchain
 instead of host

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8b28613..15b07f4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -221,7 +221,7 @@ if test "x$x86" = xyes ; then
 fi
 
 AC_MSG_CHECKING([If we are using hardfp tool chain])
-case `echo | gcc -v -xc -o - - 2>&1 | grep COLLECT_GCC_OPTIONS | tail -1` in
+case `echo | $CC -v -xc -o - - 2>&1 | grep COLLECT_GCC_OPTIONS | tail -1` in
      *float-abi=hard*) hardfp=yes; AC_MSG_RESULT([yes]) ;;
      *) hardfp=no; AC_MSG_RESULT([no]) ;;
 esac
-- 
2.7.4

