From f512355db376657ff16976803971ef411b45c259 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Tue, 3 Oct 2017 21:39:45 +0200
Subject: [PATCH 1/2] DeviceControl: Power the device on by default (temporary)

---
 devicecontrol.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/devicecontrol.cpp b/devicecontrol.cpp
index fe03d27..ce3d578 100644
--- a/devicecontrol.cpp
+++ b/devicecontrol.cpp
@@ -17,7 +17,7 @@
 #include <QtDBus/QDBusVariant>
 
 DeviceControl::DeviceControl(QObject *parent)
-:   QObject(parent), m_powered(false)
+:   QObject(parent), m_powered(true)
 {
     new DeviceAdaptor(this);
 }
-- 
2.7.4

