From 67b4066d4e1ce6c048f3b6ebf805a52a87864339 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Thu, 13 Apr 2017 11:48:18 +0200
Subject: [PATCH] Disable USB mode notifications on connect

Upstream-Status: Inappropriate

---
 src/notificationcategories/device.added.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/notificationcategories/device.added.conf b/src/notificationcategories/device.added.conf
index 2fd6df9a..c2b407d3 100644
--- a/src/notificationcategories/device.added.conf
+++ b/src/notificationcategories/device.added.conf
@@ -1,4 +1,4 @@
 x-nemo-icon=ios-usb
-urgency=2
+urgency=0
 transient=true
 x-nemo-feedback=accessory_connected
