From 7b73acdc5e10c3d4ffc0219504698bf4f489157b Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 30 Oct 2016 21:52:00 +0100
Subject: [PATCH] Add a missing math.h include

---
 src/compositor/hwcimage.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/compositor/hwcimage.cpp b/src/compositor/hwcimage.cpp
index 25ee330..24a7364 100644
--- a/src/compositor/hwcimage.cpp
+++ b/src/compositor/hwcimage.cpp
@@ -28,6 +28,8 @@
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 
+#include <math.h>
+
 static bool hwcimage_is_enabled();
 
 #define HWCIMAGE_LOAD_EVENT ((QEvent::Type) (QEvent::User + 1))
-- 
2.10.1

