From 8f46d4c2112492281d1e28b883116b804e7e21fe Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Mon, 11 May 2026 09:22:50 +0000
Subject: [PATCH] qplatforminputcontextfactory: Use qtvirtualkeyboard by
 default

Upstream-Status: Inappropriate
---
 src/gui/kernel/qplatforminputcontextfactory.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gui/kernel/qplatforminputcontextfactory.cpp b/src/gui/kernel/qplatforminputcontextfactory.cpp
index 933d990f7c9..57e8289fe0a 100644
--- a/src/gui/kernel/qplatforminputcontextfactory.cpp
+++ b/src/gui/kernel/qplatforminputcontextfactory.cpp
@@ -42,6 +42,8 @@ QStringList QPlatformInputContextFactory::requested()
     env = qgetenv("QT_IM_MODULE");
     if (!env.isEmpty())
         imList = {QString::fromLocal8Bit(env)};
+    else
+        imList = {QStringLiteral("qtvirtualkeyboard")};
 
     return imList;
 }
-- 
2.47.3

