From 6e3fd15a5e1f264b414510503ba5c5f9e82aac28 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Darrel=20Gri=C3=ABt?= <dgriet@gmail.com>
Date: Wed, 1 Feb 2023 22:26:23 +0100
Subject: [PATCH] Build kmap2qmap tool.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Darrel Griët <dgriet@gmail.com>
---
 src/src.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/src.pro b/src/src.pro
index f2675c751..9cd087e65 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -34,6 +34,8 @@ macos {
     SUBDIRS += macdeployqt
 }
 
+SUBDIRS += kmap2qmap
+
 qtHaveModule(dbus): SUBDIRS += qdbus
 
 win32|winrt:SUBDIRS += windeployqt
-- 
2.39.1

