From 0cbc9f2f1e3b25d9981cfa0471cbec498dddf83e Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 23 Oct 2016 12:05:16 +0200
Subject: [PATCH] Removes unused reference to host include directories

---
 src/src.pro | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/src.pro b/src/src.pro
index 50dd7c9..3d11252 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -33,10 +33,7 @@ DEPENDPATH += . \
     klibport \
     versit
 INCLUDEPATH += . \
-    .. \
-    /usr/include/glib-2.0 \
-    /usr/lib/glib-2.0/include \
-    /usr/include/dbus-1.0
+    ..
 
 #DEFINES += MEEGO UUID MKCAL_FOR_MEEGO TIMED_SUPPORT
 
-- 
2.9.3

