From a003e59c537ec6d84a19bedc708b6b1486b1c7c9 Mon Sep 17 00:00:00 2001
From: MagneFire <dgriet@gmail.com>
Date: Sat, 25 Dec 2021 22:40:19 +0100
Subject: [PATCH] Remove tests.

Upstream-Status: Inappropriate

---
 buteomtp.pro                                  | 35 -------------------
 .../fsstorageplugin/fsstorageplugin.pro       |  2 +-
 2 files changed, 1 insertion(+), 36 deletions(-)

diff --git a/buteomtp.pro b/buteomtp.pro
index 7ac8035..29b9ff9 100644
--- a/buteomtp.pro
+++ b/buteomtp.pro
@@ -18,46 +18,11 @@ service.depends = sub-mts
 systemd.subdir = init/systemd
 systemd.target = sub-systemd
 
-# "/opt/tests/buteo-mtp/mtp_test" - unused test application
-test.subdir = test
-test.target = sub-test
-test.depends = sub-mts
-
-# "/opt/tests/buteo-mtp/storagefactory-test" - unit test app
-mts_storage_tests.subdir = mts/platform/storage/unittests
-mts_storage_tests.target = sub-mts-storage-tests
-mts_storage_tests.depends= sub-mts
-
-# "/opt/tests/buteo-mtp/storage-test" - unit test app
-mts_fsstorage_tests.subdir = mts/platform/storage/fsstorageplugin/unittests
-mts_fsstorage_tests.target = sub-mts-fsstorage-tests
-mts_fsstorage_tests.depends = sub-mts-fsstorage-plugin
-
-# "/opt/tests/buteo-mtp/deviceinfo-test" - unit test app
-mts_deviceinfo_tests.subdir = mts/platform/deviceinfo/unittests
-mts_deviceinfo_tests.target = sub-mts-deviceinfo-tests
-mts_deviceinfo_tests.depends = sub-mts
-
-# "/opt/tests/buteo-mtp/protocol-test" - unit test app for mtp command handling
-mts_protocol_tests.subdir = mts/protocol/unittests
-mts_protocol_tests.target = sub-mts-protocol-tests
-mts_protocol_tests.depends = sub-mts
-
 SUBDIRS += \
     mts \
-    test \
-    mts_storage_tests \
     mts_fsstorage_plugin \
-    mts_fsstorage_tests \
-    mts_deviceinfo_tests \
-    mts_protocol_tests \
     service \
     systemd
 
-# install additional files for the CI Integration tests
-citests.path = /opt/tests/buteo-mtp/test-definition/
-citests.files = tests.xml
-INSTALLS += citests
-
 OTHER_FILES += rpm/buteo-mtp-qt5.privileges \
                rpm/buteo-mtp-qt5.spec
diff --git a/mts/platform/storage/fsstorageplugin/fsstorageplugin.pro b/mts/platform/storage/fsstorageplugin/fsstorageplugin.pro
index 9bd47ad..58dc076 100644
--- a/mts/platform/storage/fsstorageplugin/fsstorageplugin.pro
+++ b/mts/platform/storage/fsstorageplugin/fsstorageplugin.pro
@@ -8,7 +8,7 @@ CONFIG += plugin debug
 QT += dbus xml
 QT -= gui
 
-PKGCONFIG += blkid mount
+PKGCONFIG += blkid mount Qt5Sparql
 PKGCONFIG += nemodbus
 
 DEPENDPATH += . \
