From 6b9392cbf3bd618e78f0a5dd7e3598e00d117bc5 Mon Sep 17 00:00:00 2001
From: Florent Revest <revestflo@gmail.com>
Date: Sun, 11 Feb 2024 15:45:01 +0100
Subject: [PATCH] Advertise hostname in BLE ad payload to have a recognizable

 name in AsteroidOSSync

Upstream-Status: Inappropriate [AsteroidOS specific]

---
 src/shared/ad.c | 30 ++++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/src/shared/ad.c b/src/shared/ad.c
index 951c56c60..4dbdd8e98 100644
--- a/src/shared/ad.c
+++ b/src/shared/ad.c
@@ -414,6 +414,18 @@ static bool ad_replace_data(struct bt_ad *ad, uint8_t type, const void *data,
 	return false;
 }
 
+static size_t local_name_length()
+{
+	size_t length = 0;
+	char hostname[8];
+
+	length += 2; /* one byte for length and one byte for 0x09 */
+	gethostname(hostname, 8);
+	length += strnlen(&hostname, 8);
+
+	return length;
+}
+
 static size_t uuid_list_length(struct queue *uuid_queue)
 {
 	bool uuid16_included = false;
@@ -524,6 +536,8 @@ static size_t calculate_length(struct bt_ad *ad)
 {
 	size_t length = 0;
 
+	length += local_name_length();
+
 	length += uuid_list_length(ad->service_uuids);
 
 	length += uuid_list_length(ad->solicit_uuids);
@@ -541,6 +555,20 @@ static size_t calculate_length(struct bt_ad *ad)
 	return length;
 }
 
+static void serialize_local_name(struct iovec *iov)
+{
+	char hostname[8];
+	uint8_t hostname_len;
+
+	gethostname(hostname, 8);
+	hostname_len = strnlen(&hostname, 8);
+
+	util_iov_push_u8(iov, hostname_len + 1);
+	util_iov_push_u8(iov, EIR_NAME_COMPLETE);
+
+	util_iov_push_mem(iov, hostname_len, &hostname);
+}
+
 static void serialize_uuids(struct queue *uuids, uint8_t uuid_type,
 				uint8_t ad_type, struct iovec *iov)
 {
@@ -709,6 +737,8 @@ uint8_t *bt_ad_generate(struct bt_ad *ad, size_t *length)
 
 	iov.iov_len = 0;
 
+	serialize_local_name(&iov);
+
 	serialize_service_uuids(ad->service_uuids, &iov);
 
 	serialize_solicit_uuids(ad->solicit_uuids, &iov);
-- 
2.43.0

